### ISSUE TITLE
Exam answer sheet error for 4.A

### ISSUE THREAD

**Issue by AnnMarieDavis** [Section508Coordinators/DHSA-TT-200-A#386]
## Issue Type 
- [X] Incorrect content

## Description
Page 3 - Bottom banner Register Your Device no focus
 
answer sheet for the Fail for 4.A  indicates the Data and security tab as the answer.  this is the link AFTER Register Your Device
![4 A answer sheet](https://user-images.githubusercontent.com/31414225/71373603-cb4e3600-2585-11ea-8113-36f567e8930c.png)

![4 A test page](https://user-images.githubusercontent.com/31414225/71373656-eb7df500-2585-11ea-97d5-a3727d61125b.png)

## Location
https://exams.section508testing.net/dhsv5/scormapi/admin.php?SCOInstanceID=14558&KeyID=96554&ExamID=2&studentEmail=mike@studioefx.com

https://exams.section508testing.net/dhsv5/dhscourse/Lesson1/exams/web-practice/testPages/page3_main.php?studentEmail=annmarie.davis@associates.hq.dhs.gov&SCOInstanceID=606&examId=2 

## Expected Behavior / Recommendation
answer sheet change to correct the Fail for 4.A  -- Register Your Device does not receive focus so the preceding tab About Registration should be the correct answer

**Comment by mikebarklage** [comment]
Updated on Dev.

**Comment by Tonylayton1** [comment]
Not sure what was changed. When the second footer link has no visual focus, it fails 4.D, not 4.A. even though there is no visual focus, it can still be accessed with the keyboard, so it passes 4.A.

4.D was already testing correctly. When this link failed to show visual focus, the preceding link is the correct answer (r5_1)

To clear up some confusion, there is a randomized 4.A failure on the third header link (as opposed to the footer link), which is "data and Security" found in region 1. (r1_6). 4.A however does not specify that anything other than the inaccessible control itself as the correct answer. 


To summarize: I'm not sure what changed, but nothing should have changed.

**Comment by mikebarklage** [comment]
To clarify what I updated, I changed the 4.A failure on that page from r1_6 to r1_4. If that's incorrect, it's easy to change it back.

**Comment by Tonylayton1** [comment]
yes, 4.A should remain 1_6.
4.A and 4.D were confused here, and header vs footer links were also confused. On the selection list it is clear that an item is in the footer vs the header, but in the admin tool the links in the header and footer look the same.

**Comment by mikebarklage** [comment]
Yeah, I just doublechecked the code and you're right. I've changed it back to r1_6.

**Comment by Tonylayton1** [comment]
that's been verified. moving to DHS review - please let me know if my assessment above is incorrect and that there needs to be a change here.

**Comment by AnnMarieDavis** [comment]
I'm the one who lobbied for marking the element that LAST received correct focus, but I STILL goof and mark the one that doesn't receive focus . This is correctly marking 4.D and the message is correct.
